paper-fab-speed-dial.html

```diff
@@ -66,36 +66,60 @@
 
 <script>
 
-(function() {
-
-	Polymer({
-		is: 'paper-fab-speed-dial',
-		properties: {
-			opened: {
-				type: Boolean,
-				notify: true
-			}
-		},
-
-		// Public methods
-		open: function(e) {
-			// Required for mobile Safari to avoid passing the tap event to an element below the FAB
-			if (e) {
-				e.preventDefault();
+	(function() {
```

Comment:
Could you please remove the tab intend, so the diff of your PR contains only code that reverse to the backdrop issue?